installing/install_config/customizations.adoc

```diff
@@ -29,46 +29,55 @@ Custom Resource Definitions (CRDs) that are deployed on your cluster.
 |Configure the Kubernetes API server.
 
 |Authentication
-|You can configure the identity provider for your cluster.
+|You can configure the
+xref:../../authentication/understanding-identity-provider.adoc#understanding-identity-provider[identity provider]
+for your cluster.
 
 |Build
 |Configure build settings.
 
 |ClusterVersion
-|Configure the desired version of the cluster and how updates are handled.
+|In {product-title} {product-version}, you must not customize the ClusterVersion
+resource for production clusters. Instead, follow the process to
+xref:../../updating/updating-cluster.adoc#updating-cluster[update a cluster].
 
 |Console
 |Configure the web console.
 
 |DNS
-|Configure DNS.
+|To configure DNS by using the Network Operator, you must
```

Comment:
The CNO doesn't manage DNS. We have the following for the DNS Operator itself, but it doesn't cover configuration:

https://docs.openshift.com/container-platform/4.1/networking/dns/dns-operator.html

Reply:
Looking at https://github.com/openshift/api/blob/master/operator/v1/types_dns.go,
The DNS ClusterOperator global configuration only reports status.